content/locations/states.yml

```diff
@@ -0,0 +1,299 @@
+state: Alabama
```

Comment:
Or it could be all at the top level:
```
Alabama:
- url: http://www.acjic.alabama.gov/
  text: Alabama Criminal Justice Information Center
- url:
  text:
```

The problem is that right now there are a bunch of `state` keys at the same level and I think YAML doesn't like that